packages/hub/src/graphql/types/ModelRevision.ts

```diff
@@ -9,11 +11,25 @@ export const SquiggleSnippet = builder.prismaNode("SquiggleSnippet", {
   fields: (t) => ({
     code: t.exposeString("code"),
     version: t.exposeString("version"),
+    seed: t.exposeString("seed"),
   }),
 });
 
 // TODO - turn into interface?
-export const ModelContent = builder.unionType("ModelContent", {
+export const ModelContent: UnionRef<
+  {
+    id: string;
+    code: string;
+    version: string;
+    seed: string;
+  },
+  {
+    id: string;
+    code: string;
+    version: string;
+    seed: string;
+  }
```

Comment:
I've investigated a bit, and this should be reverted.

My guess is that you got a "likely not portable" error in VS Code that recommended a fix, but:
- this error is not new, it also exists in main
- it's VS Code-only; `tsc` in CLI doesn't complain
- there are ~10 places with a similar error
- it's a bug in TypeScript; details here: https://github.com/microsoft/TypeScript/issues/42873
- I've fixed this in main: https://github.com/quantified-uncertainty/squiggle/commit/d26c0ec6d5da7918959eb16f59ed93c2240159fd (safe to commit directly because TypeScript configuration is only used for checks and won't affect Next.js deployments)

Reply:
yep, that's what happened. Happy to revert. Thanks for investigating!